config/apidoc.php

```diff
@@ -50,6 +50,34 @@
      */
     'routes' => [
         [
+            /*
+             * Group of settings for managing routes in Laravel Framework.
+             */
+
+            'laravel' => [
```

Comment:
Good call to nest under a `laravel` key, but this should not be inside `routes`. The `routes` key is explicitly for defining route groups that documentation should be generated for. Just put it at the top-level, after `type`.

Reply:
Fixed